apps/nextjs-website/src/_contents/webinars.ts

```diff
@@ -2,6 +2,28 @@ import { Webinar } from '@/lib/types/webinar';
 import { appIOGuideListsPath } from './appIo/guideListsPath';
 import { appIoQuickStartGuidePath } from './appIo/quickStartGuidePath';
 
+function imageFromPath(imagePath: string): {
+  readonly name: string;
+  readonly alternativeText: string | null;
+  readonly width: number;
+  readonly height: number;
+  readonly ext: string;
+  readonly mime: string;
+  readonly url: string;
+} {
+  // eslint-disable-next-line functional/immutable-data
+  const ext = imagePath.split('.').pop();
+  return {
+    name: imagePath,
+    alternativeText: null,
+    width: 400,
+    height: 400,
+    ext: `.${ext}`,
+    mime: `image/${ext}`,
+    url: imagePath,
+  };
+}
+
```

Comment:
Is this the correct file to put this function into? Would it be better to move to an helper file?

Reply:
I think is better to leave it here beacuse this function is only used here and will be removed along with this file once webinars will be completed moved in CMS.